resources/gbxml/TestSchedules.xml

```diff
@@ -0,0 +1,164 @@
+    <ScheduleValue>0.5</ScheduleValue>
+    <ScheduleValue>0.3</ScheduleValue>
+    <ScheduleValue>0.3</ScheduleValue>
+    <ScheduleValue>0.2</ScheduleValue>
+    <ScheduleValue>0.2</ScheduleValue>
+    <ScheduleValue>0</ScheduleValue>
+    <ScheduleValue>0</ScheduleValue>
+    <Name>Complex Year Schedule - Typical Day January to June</Name>
+  </DaySchedule>
+
+  <WeekSchedule type="Fraction" id="Complex_Schedule_WeekSchedule_EndYear">
+    <Day dayScheduleIdRef="Complex_Schedule_DaySchedule_EndYear_Weekday" dayType="Weekday" />
+    <Day dayScheduleIdRef="Complex_Schedule_DaySchedule_EndYear_Weekend" dayType="Weekend" />
+    <Name>Complex Year Schedule - Typical Week July to December</Name>
+  </WeekSchedule>
+  <DaySchedule type="Fraction" id="Complex_Schedule_DaySchedule_EndYear_Weekday">
+    <ScheduleValue>0</ScheduleValue>
+    <ScheduleValue>0</ScheduleValue>
+    <ScheduleValue>0</ScheduleValue>
+    <ScheduleValue>0</ScheduleValue>
+    <ScheduleValue>0</ScheduleValue>
+    <ScheduleValue>0</ScheduleValue>
+    <ScheduleValue>0.5</ScheduleValue>
+    <ScheduleValue>0.5</ScheduleValue>
+    <ScheduleValue>0.5</ScheduleValue>
+    <ScheduleValue>0.5</ScheduleValue>
+    <ScheduleValue>0.5</ScheduleValue>
+    <ScheduleValue>0.5</ScheduleValue>
+    <ScheduleValue>0.5</ScheduleValue>
+    <ScheduleValue>0.5</ScheduleValue>
+    <ScheduleValue>0.5</ScheduleValue>
+    <ScheduleValue>0.5</ScheduleValue>
+    <ScheduleValue>0.5</ScheduleValue>
+    <ScheduleValue>0.5</ScheduleValue>
+    <ScheduleValue>0.5</ScheduleValue>
+    <ScheduleValue>0.5</ScheduleValue>
+    <ScheduleValue>0.5</ScheduleValue>
+    <ScheduleValue>0.5</ScheduleValue>
+    <ScheduleValue>0</ScheduleValue>
+    <ScheduleValue>0</ScheduleValue>
+    <Name>Complex Year Schedule - Typical Weekday July to December</Name>
+  </DaySchedule>
+  <DaySchedule type="Fraction" id="Complex_Schedule_DaySchedule_EndYear_Weekend">
+    <ScheduleValue>0</ScheduleValue>
+    <ScheduleValue>0</ScheduleValue>
+    <ScheduleValue>0</ScheduleValue>
+    <ScheduleValue>0</ScheduleValue>
+    <ScheduleValue>0</ScheduleValue>
+    <ScheduleValue>0</ScheduleValue>
+    <ScheduleValue>1.0</ScheduleValue>
+    <ScheduleValue>1.0</ScheduleValue>
+    <ScheduleValue>1.0</ScheduleValue>
+    <ScheduleValue>1.0</ScheduleValue>
+    <ScheduleValue>1.0</ScheduleValue>
+    <ScheduleValue>1.0</ScheduleValue>
+    <ScheduleValue>1.0</ScheduleValue>
+    <ScheduleValue>1.0</ScheduleValue>
+    <ScheduleValue>1.0</ScheduleValue>
+    <ScheduleValue>1.0</ScheduleValue>
+    <ScheduleValue>1.0</ScheduleValue>
+    <ScheduleValue>1.0</ScheduleValue>
+    <ScheduleValue>1.0</ScheduleValue>
+    <ScheduleValue>1.0</ScheduleValue>
+    <ScheduleValue>1.0</ScheduleValue>
+    <ScheduleValue>1.0</ScheduleValue>
+    <ScheduleValue>0</ScheduleValue>
+    <ScheduleValue>0</ScheduleValue>
+    <Name>Complex Year Schedule - Typical Weekend July to December</Name>
+  </DaySchedule>
```

Comment:
A more complex schedule: two separate periods

* Jan to June, same day for all
* July to December, one day schedule for weekday and one for weekends